crates/re_viewer/src/misc/transform_cache.rs

```diff
@@ -239,10 +239,8 @@ fn transform_at(
                     // A pinhole camera means that we're looking at an image.
                     // Images are spanned in their local x/y space.
                     // Center it and move it along z, scaling the further we move.
-
-                    let distance = entity_properties
-                        .get(entity_path)
-                        .pinhole_image_plane_distance(&pinhole);
+                    let props = entity_properties.get(entity_path);
+                    let distance: f32 = (*props.pinhole_image_plane_distance.get()).into();
```

Comment:
Probably can get away with
```suggestion
                    let distance = props.pinhole_image_plane_distance.get().into_inner();